src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -1107,7 +1107,8 @@
         "3.0.0",
         "5.0.0",
         "5.0.1",
-        "5.0.2"
+        "5.0.2",
+        "5.0.3"
       ],
       "BaselineVersion": "5.0.2",
```

Comment:
this needs to be updated as well

Reply:
Good point. I rebased it on release/5.0, which has picked up this (and other) changes from https://github.com/dotnet/runtime/pull/57226